ledger/src/blockstore.rs

```diff
@@ -5384,12 +5445,13 @@ pub mod tests {
                 assert_eq!(meta.parent_slot, Some(slot - 1));
             }
             assert_eq!(meta.last_index, Some(shreds_per_slot as u64 - 1));
-            assert!(meta.is_connected);
+            assert!(meta.is_connected());
         }
     }
 
     #[test]
     fn test_handle_chaining_missing_slots() {
```

Comment:
Let's add some description about this test, specifically on why it expects is_parent_connected is true while is_connected is false.